test/helpers/desk_helper_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class DeskHelperTest < ActionView::TestCase
+
+  setup do
+    @occupied_desk = desks :desk1
+    @unoccupied_desk = desks :desk2
+  end
+
+  test '#desk_status gives correct status' do
```

Comment:
Styling nit that I really don't feel strong about whatsoever: In rails, test names are usually wrapped in double quotes

Reply:
Was following the test cases in models/desk_test.rb, should this file be changed to include double quotes as well?

Reply:
Nah that's ok 🙂